internal/openshift/controller.go

```diff
@@ -89,7 +89,7 @@ func (c *controllerImpl) HandleBuild(o model.Object) error {
 	sendUserToIdler := false
 	userIdler := c.userIdlerForNamespace(ns)
 	user := userIdler.GetUser()
-	if c.isActive(&o.Object) {
+	if c.isActive(&o.Object) && o.Type != "DELETED" {
```

Comment:
shoudn't we change `isActive` instead to return false if `o.Type == "DELETED"` ?